Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -92,7 +92,20 @@ public void ResetConfiguration(MixedRealityConfigurationProfile profile)
 
         private int mixedRealityComponentsCount = 0;
 
-        #endregion Mixed Reality Manager Profile configuration
+        #endregion Mixed Reality runtime component registry
+
+        #region Active SDK components
+
+        /// <summary>
+        /// The Active Device property maintains the SDK detected by the Mixed Reality manager on startup
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The currently active / detected Headset or SDK")]
+        private IMixedRealityDevice activeDevice = default(IMixedRealityDevice);
```

Comment:
Is it possible to have more than one active at a time?

Reply:
This is scaffold only as we haven't got the SDK detection stuff in there yet
In reality, NO.  As you can only have one headset on at a time :D (unless Zaphod Beeblebrox wants to try it)